README.md

```diff
@@ -593,15 +593,17 @@ that are common to all storage systems are omitted for brevity.
 | **Features**                | Gaia | [Sia](https://sia.tech/) | [Storj](https://storj.io/) | [IPFS](https://ipfs.io) | [DAT](https://datproject.org) | [SSB](https://www.scuttlebutt.nz/) |
 |-----------------------------|------|--------------------------|----------------------------|-------------------------|-------------------------------|------------------------------------|
 | User controls where data is hosted           | X |   |   |   |   |   |
-| Data can be viewed in a normal Web browser   | X |   |   | X |   |   |
+| Data stored redundantly                      | optional | X | X | optional | optional | optional |
 | Data is read/write                           | X |   |   |   | X | X |
-| Data can be deleted                          | X |   |   |   | X | X |
+| Data can be deleted                          | X | X |   |   | X | X |
```

Comment:
In my mind, "deleted" means "once the request is acknowledged, a subsequent read will always fail."  Is this true for Sia?  Or does Sia just garbage-collect erasure codes in the background?  If it's the latter, then I don't consider this to be "deleted."

Reply:
Correct. At this time it is garbage-collection, if the contract is terminated, deleted data will remain on the storage node until the contract expires. "Delete" function is being worked on by the Sia team. Estimate is a few months until deployment. Theory behind this is complex but once delete is implemented it will be deleted node-side.

Reply:
Ah, good to know.  Thank you for the clarification!